README.md

```diff
@@ -1,11 +1,16 @@
- ## **Environment Variables**
-API_KEY: Telegram bot API Key
-CLIENT_ID: Amadeus API Client ID
-CLIENT_SECRET: Amadeus API Client Secret Key
+# AirTix
+AirTix is a Telegram bot that helps the user look for the cheapest air tickets for a specific duration.
 
+## **Environment Variables**
 | Variable | Data type | Default value | Description  |
 | ---------- | --------- | -------------------------- | -----|
 | API_KEY | String | '' | Telegram Bot API Key |
 | CLIENT_ID | String | '' | Amadeus Client Keys |
 | CLIENT_SECRET | String | '' | Amadeus Client Secret Keys |
```

Comment:
Still using amadeus? If not, can remove. If the variables are being used currently, but updated in another branch (i.e this branch need amadeus, but the feat branch no need alr) then update the readme in the new branch instead